client.go

```diff
@@ -74,16 +77,22 @@ func NewClient(account, token string) *Client {
 	c := http.DefaultClient
 	c.Timeout = 15 * time.Second
 	baseURLString := fmt.Sprintf("https://%s.tpondemand.com/api/v1/", account)
+	baseURLReadOnlyString := fmt.Sprintf("https://%s.tpondemand.com/api/v2/", account)
 	baseURL, err := url.Parse(baseURLString)
 	if err != nil {
 		panic(err)
 	}
+	baseURLReadOnly, err := url.Parse(baseURLReadOnlyString)
+	if err != nil {
+		panic(err)
```

Comment:
I know this is what we had before, but is `panic` correct here? Should we be returning an error instead?

Reply:
I think you're correct about this. I'll fix it up.